.gitignore

```diff
@@ -339,3 +339,6 @@ ASALocalRun/
 /src/Simulation/Simulators.Tests/TestProjects/QSharpExe/built
 /src/Simulation/Simulators.Tests/TestProjects/TargetedExe/built
 dbw_test
+
+# Generated Q# docs
+**/docs
```

Comment:
Is there a difference between `**/docs` and just `docs`?

Reply:
Now that you mention it, no there really isn't. I could change this to `[Dd]ocs` and but it up next to the `[Bb]in` and `[Oo]bj` exclusions for clarity.